languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/added.txt

```diff
@@ -1218,4 +1218,17 @@ tripé	triper	V ppa m s
 tripée	triper	V ppa f s
 tripées	triper	V ppa f p
 tripés	triper	V ppa m p
-Apple	Apple	Z e sp
\ No newline at end of file
+Apple	Apple	Z e sp
+postsoviétique	postsoviétique	J e sp
+postromantique	postromantique	N e sp
+postromantique	postromantique	J e sp
+postdoc	postdoc	J e sp
+postélectoral	postélectoral	J m s
+postélectorale	postélectoral	J f s
+postélectorales	postélectoral	J f p
+postélectoraux	postélectoral	J m p
+postimpressionniste	postimpressionniste	J e sp
+postimpressionniste	postimpressionniste	N e sp
+postimpressionnisme	postimpressionnisme	N e sp
+postuniversitaire	postuniversitaire	J e s
+postuniversitaires	postuniversitaire	J e p
```

Comment:
Are these plural forms correct? Shouldn't they be with 's'? postsoviétiques, postromantiques...

Reply:
Yes, apparently there are correct. I added them to the file.